src/ui/profile_preferences.ui

```diff
@@ -5306,9 +5443,6 @@
   <tabstop>comboBox</tabstop>
   <tabstop>mMapperUseAntiAlias</tabstop>
   <tabstop>checkBox_showDefaultArea</tabstop>
-  <tabstop>fontComboBox_mapSymbols</tabstop>
```

Comment:
Please put these back in the right place - though of course changes to this form will of course clash with PR #3929 .